lib/pwauth/okta/impl.go

```diff
@@ -5,35 +5,74 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"time"
 
 	"github.com/Cloud-Foundations/Dominator/lib/log"
 )
 
 const (
-	authPath           = "/api/v1/authn"
-	authEndpointFormat = "https://%s.okta.com" + authPath
+	authPath               = "/api/v1/authn"
+	authEndpointFormat     = "https://%s.okta.com" + authPath
+	factorsVerifyPathExtra = "/factors/%s/verify"
 )
 
```

Comment:
Please unexport all types.
Please unexport all fields except those which need to be written by the JSON decoder.